src/Components/Footer/styles.css

```diff
@@ -0,0 +1,55 @@
+.footer {
+  padding: var(--gap-medium);
+  background-color: var(--background-color);
+  color: rgb(240, 232, 232);
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+}
+
+.footer-main {
+  display: flex;
+  flex: 1 1 0px;
+  justify-content: space-around;
+}
+.footer-main li {
+  margin: 1rem 0;
+}
+
+.footer-main li a {
+  color: white;
+}
```

Comment:
you can use variable naming for color

Reply:
I use them but only for black and white I use normal color names